README.md

```diff
@@ -40,12 +40,14 @@ The two main use-cases are:
 
 The PreLiquidation smart-contract enforces the properties:
 
-- preLltv < LLTV;
-- preLCF1 <= preLCF2;
-- preLFC1 <= 1;
-- 1 <= preLIF1 <= preLIF2 <= 1 / LLTV.
-  Note: using `preLCF2 > 1`, one can select at which LTV between preLltv and LLTV the entire position can be pre-liquidated.
-  A pre-liquidation close factor higher than 100% means that the whole position is pre-liquidatable.
+- `preLltv < LLTV`;
+- `preLCF1 <= preLCF2`;
+- `preLFC1 <= 1`;
+- `1 <= preLIF1 <= preLIF2 <= 1 / LLTV`.
+
+Note: A pre-liquidation close factor higher than 100% means that the whole position is pre-liquidatable, so using `preLCF2 > 1` allows to make the position entirely pre-liquidatable at a LTV between preLltv and LLTV.
+
+The property `preLIF2 <= 1 / LLTV` ensures that pre-liquidation always improves the health of the position.
```

Comment:
This is not the only assumption that is needed: health doesn't get worse iff `preLIF <= 1 / LTV`
This is ensured because pre-liquidations are only allowed for `LTV <= LLTV`, which implies that `1 / LLTV <= 1 / LTV`. So together with the property `preLIF2 <= 1 / LLTV` (and `preLIF <= preLIF2`) we get `preLIF <= 1 / LTV`
```suggestion
The property `preLIF2 <= 1 / LLTV` and the fact that pre-liquidations are only allowed for `LTV <= LLTV` ensure that pre-liquidations can't worsen the health of the position.
```